### ISSUE TITLE
Intel App

### ISSUE THREAD

**Issue by Notorious0830** [landonepps/op1.fun.app#4]
Hello,

Thank you for the effort in updating the app appreciate the work. Using Intel Mac and get the same problem trying to log in. Putting in the correct log-in information and when clicking log in it doesn’t do anything. Do I need to edit the Index.js after installing your new updated version. Using the X64 for Intel. Put this same comment in the OP1 Fun Forums.

**Comment by alexgazquez** [comment]
Yes, the website and this app are truly wonderful. However, some of us can't enjoy it... It happens to me too. I have a 2017 Intel Mac and I can't install the app because it says that Apple can't check for malicious software. Hopefully the software developers can fix it... Since I'm a chemist
